### PR TITLE
[REVIEW] Added VCSR format into hugectr model.py for NVTabular + HugeCTR ensemble

### PR DESCRIPTION
**Pull request by oyilmaz-nvidia**

### CODE REVIEW

File: nvtabular/inference/triton/model_hugectr.py [hunk 1]
```diff
@@ -83,16 +83,17 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
...
-                        _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32),
+                        cats_np,
                     )
                 )
             else:
                 output_tensors.append(Tensor("CATCOLUMN", np.array([0])))
 
-            row_index = [i for i in range(len(self.workflow.column_group.input_column_names))]
+            row_index = [i for i in range(len(cats_np))]
             row_index = np.array(row_index).astype(np.int32)
```
Comment (benfred) [inline nvtabular/inference/triton/model_hugectr.py:97]: We can do this without creating a python list in pure numpy 
```suggestion
            row_index = np.arange(len(cats_np), dtype=np.int32)
```

File: nvtabular/inference/triton/model_hugectr.py [hunk 2]
```diff
@@ -83,16 +83,17 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
                 output_tensors.append(Tensor("DES", np.array([0])))
 
             if "cats" in self.column_types:
+                cats_np = _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32)
                 output_tensors.append(
                     Tensor(
                         "CATCOLUMN",
-                        _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32),
+                        cats_np,
```
Comment (benfred) [inline nvtabular/inference/triton/model_hugectr.py:90]: Can you run black on this? its failing CI because of the formatting -

Comment (oyilmaz-nvidia) [reply nvtabular/inference/triton/model_hugectr.py:90]: Will do! Keep forgetting it.

**Comment by oyilmaz-nvidia** [comment]
Let's not merge this PR until next Monday-Tuesday since I need to test it in a ensemble model.

**Review by benfred — APPROVED** [review]

**Pull request merged**